Formula/reprepro.rb

```diff
@@ -0,0 +1,37 @@
+class Reprepro < Formula
+  desc "Debian package repository manager"
+  homepage "https://salsa.debian.org/brlink/reprepro"
+
+  url "https://deb.debian.org/debian/pool/main/r/reprepro/reprepro_5.3.0.orig.tar.gz"
+  sha256 "5a5404114b43a2d4ca1f8960228b1db32c41fb55de1996f62bc1b36001f3fab4"
+
+  depends_on "berkeley-db@4"
```

Comment:
Does this not support the latest `berkeley-db`?

Reply:
Nope, there was a backwards-incompatible change of the on-disk format used by BDB 4 vs 5+:

https://stackoverflow.com/questions/49354902/berkeley-db-4-and-later-versions-compatibility

There's still a few major OSS projects that are stuck with BDB 4; the most notable one in Homebrew is probably [`bitcoin`](https://github.com/Homebrew/homebrew-core/blob/master/Formula/bitcoin.rb#L23).